modules/nexus/packer/ami/packer.json

```diff
@@ -0,0 +1,112 @@
+                ],
+                "most_recent": true
+            },
+            "ssh_username": "ubuntu",
+            "run_tags": {
+                "Name": "{{user `ami_base_name` }}-{{isotime | clean_ami_name}}",
+                "Base Name": "{{user `ami_base_name` }}",
+                "Timestamp": "{{isotime \"2006-01-02 03:04:05\"}}",
+                "Packer": "yes",
+                "Consul Version": "{{user `consul_version` }}",
+                "Nexus Version": "{{user `nexus_version` }}"
+            },
+            "tags": {
+                "Name": "{{user `ami_base_name` }}-{{isotime | clean_ami_name}}",
+                "Base Name": "{{user `ami_base_name` }}",
+                "Timestamp": "{{isotime \"2006-01-02 03:04:05\"}}",
+                "Packer": "yes",
+                "Consul Version": "{{user `consul_version` }}",
+                "Nexus Version": "{{user `nexus_version` }}"
+            },
+            "snapshot_tags": {
+                "Name": "{{user `ami_base_name` }}-{{isotime | clean_ami_name}}",
+                "Base Name": "{{user `ami_base_name` }}",
+                "Timestamp": "{{isotime \"2006-01-02 03:04:05\"}}",
+                "Packer": "yes",
+                "Consul Version": "{{user `consul_version` }}",
+                "Nexus Version": "{{user `nexus_version` }}"
+            }
+        }
+    ],
+    "provisioners": [
+        {
+            "type": "shell",
+            "inline": [
+                "timeout 60s bash -c \"while ! [ -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting on cloud-init...'; sleep 2; done\""
+            ]
+        },
+        {
+            "type": "ansible",
+            "playbook_file": "{{ template_dir }}/site.yml",
+            "user": "ubuntu",
+            "extra_arguments": [
+                "-e",
+                "{ \"additional_ntp_servers\": {{user `additional_ntp_servers`}} }",
+                "-e",
+                "consul_module_version={{user `consul_module_version`}} consul_version={{user `consul_version`}} consul_module_repo={{user `consul_module_repo`}}",
+                "-e",
+                "{ \"consul_enable_syslog\": {{user `consul_enable_syslog`}} }",
+                "-e",
+                "td_agent_config_file={{user `td_agent_config_file`}} td_agent_config_vars_file={{user `td_agent_config_vars_file`}}",
+                "-e",
+                "ca_certificate={{user `ca_certificate`}}",
+                "-e",
+                "consul_host={{user `consul_host`}} consul_port={{user `consul_port`}} consul_scheme={{user `consul_scheme`}} consul_token={{user `consul_token`}} consul_integration_prefix={{user `consul_integration_prefix`}}",
+                "-e",
+                "nexus_version={{user `nexus_version`}} nexus_db_dir={{user `nexus_db_dir`}}",
+                "-e",
+                "timezone={{user `timezone`}}",
+                "-e",
+                "ansible_python_interpreter=/usr/bin/python3"
```

Comment:
Guess you could try this too? https://github.com/datagovsg/terraform-modules/pull/233/files#diff-6ab1fec7168f06f35842e38b6ae11cf9R99